src/i18n/locales/en/default.json

```diff
@@ -280,6 +280,19 @@
         "button": "Start for free"
       }
     },
+    "emailVerify": {
+      "greeting": "Hey {{name}},",
+      "description": "your account is not verified, vocascan-server has send you an email. Please verify your email address by clicking on the link provided in the email.",
```

Comment:
Write the first letters in uppercase.

Reply:
I thought starting lowercase because the greeting line ends with a `,` should I change this?